packages/tomcat.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Install tomcat locally - https://tomcat.apache.org/index.html
+#
+# change the following environment variables to your project configuration as needed,
+# otherwise the defaults below will be used.
+# * TOMCAT_VERSION
+#
+# Include the following command your builds:
+# \curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/tomcat.sh | bash -s
+
+TOMCAT_VERSION=${TOMCAT_VERSION:="8.5.9"}
+TOMCAT_DIR=${TOMCAT_DIR="$HOME/tocat"}
+TOMCAT_WAIT_TIME=${TOMCAT_WAIT_TIME:="10"}
+
+set -e
+CACHED_DOWNLOAD="${HOME}/cache/apache-tomcat-${TOMCAT_VERSION}.tar.gz"
+
+mkdir -p "${TOMCAT_DIR}"
+wget --continue --output-document "${CACHED_DOWNLOAD}" "http://www-us.apache.org/dist/tomcat/tomcat-8/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz"
```

Comment:
I just noticed, that the URL includes the major version on the directory level as well. An example of how to parse the major version from a variable is available at https://github.com/codeship/scripts/blob/master/packages/elasticsearch.sh#L19